collect_app/src/main/res/xml/preferences.xml

```diff
@@ -75,6 +75,12 @@
             android:key="font_size"
             android:title="@string/font_size" />
         <CheckBoxPreference
+            android:id="@+id/instance_sync"
+            android:defaultValue="true"
+            android:key="instance_sync"
```

Comment:
Yes we can do that. But I noticed that all of the other keys for the preferences were hardcoded strings too. So just to keep the general pattern of the preferences I used the hardcoded string too